### ISSUE TITLE
Empty Roles

### ISSUE THREAD

**Issue by nullatonce** [DevMarketer/DevMarketer#53]
When passing empty roles in UserController -> update it returns erro, how to chande that, so i can pass empty roles value ?

**Comment by mjcarter95** [comment]
I don't think you should have no roles assigned to a user; they should at least have the "subscriber" role which is the base role with the least rights.
